test/convolutional_test.py

```diff
@@ -192,6 +192,63 @@ def test_mstgcn():
     assert outputs1.shape == (batch_size, node_count, num_for_predict)
     assert outputs2.shape == (batch_size, node_count, num_for_predict)
 
+def test_mtgnn():
+    """
+    Testing MTGNN block
+    """
+    gcn_true = True
+    buildA_true = True
+    dropout = 0.3
+    subgraph_size = 20
+    gcn_depth = 2
+    num_nodes = 207
+    node_dim = 40
+    dilation_exponential = 1
+    conv_channels = 32
+    residual_channels = 32
+    skip_channels = 64
+    end_channels = 128
+    in_dim = 2
+    seq_in_len = 12
+    seq_out_len = 10
+    layers = 3
+    batch_size = 16
+    propalpha = 0.05
+    tanhalpha = 3
+    num_split = 1
+
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    _, edge_index = create_mock_data(number_of_nodes=num_nodes, edge_per_node=8, in_channels=in_dim)
```

Comment:
Why do you need this? Why edge index is not good?

Reply:
This is to create a predefined adjacency matrix.

Reply:
Can we do it in purely Pytorch?

Reply:
No problem